Library/Homebrew/formula.rb

```diff
@@ -459,6 +465,13 @@ def bottle_for_tag(tag = nil)
   # @see .version
   delegate version: :active_spec
 
+  # Stop RuboCop from erroneously indenting hash target
+  delegate [ # rubocop:disable Layout/HashAlignment
+    :allow_network_access,
+    :deny_network_access,
```

Comment:
```suggestion
    :allow_network_access!,
    :deny_network_access!,
```

for method names maybe (based on https://github.com/Homebrew/homebrew-core/pull/169720). happy to be overruled if you/others disagree!

Reply:
+1 from me on adding `!` -- IMO it makes it a bit clearer that this mutates the global "environment" the formula's stages operate in 🙂